2-advanced/dubbo-samples-cache/pom.xml

```diff
@@ -36,12 +36,12 @@
     <description>Dubbo Samples Cache</description>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <dubbo.version>3.2.6</dubbo.version>
-        <spring-boot.version>2.7.8</spring-boot.version>
+        <dubbo.version>3.3.0-beta.1</dubbo.version>
```

Comment:
```suggestion
        <dubbo.version>3.2.11</dubbo.version>
```

Reply:
Must use `<dubbo.version>3.3.0-beta.1</dubbo.version>` in order to showcase usage of `dubbo-zookeeper-curator5-spring-boot-starter`